changelog.d/pa-1943.fixed

```diff
@@ -0,0 +1 @@
+JS/TS: Allowed parsing of patterns using the optional chaining "?." syntax.
```

Comment:
This should be `gh-6201.fixed`, we should always use the GitHub issue if it exists. Users don't have access to our Linear board.